File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        int longest = 0;
        for(int i=0; i < nums.size(); i++)
        {
            int size = 1;
            bool valid = false;
            int idx = i;
            do
            {
                for(int j=0; j < nums.size(); j++)
                {
                    if(nums[j] == nums[idx] + 1)
                    {
                        size += 1;
                        valid = true;
                        idx = j;
                        break;
                    }
                    else
                    {
                        valid = false;
                    }
                }

            }while(valid);

            if(size > longest)
            {
                longest = size;
            }
        }

        return longest;
    }
};

File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-0.py
class Solution:
    def findMin(self, nums: List[int]) -> int:
        l,r = 0,len(nums) - 1

        minimum = nums[l]

        while l <= r: 
            if nums[r] < minimum: 
                minimum = nums[r]
            
            if nums[r] < nums[l]:
                l+=1
            else:
                r-=1

            if nums[l] < minimum:
                minimum = nums[l]
        
        return minimum

            
            